### ISSUE TITLE
Installation fails due to conflicting wheel version

### ISSUE THREAD

**Issue by NeolithEra** [pythonzm/Ops#25]
Hi, users are unable to run ops due to dependency conflict with _**wheel**_ package. As shown in the following full dependency graph of **_ops_**, **_billiard==3.5.0.4_** requires _**wheel>=0.29.0**_，while django-crispy-forms==1.7.2 requires _**wheel==0.24.0**_.

According to pip’s “first found wins” installation strategy, _**wheel==0.33.4**_ is the actually installed version. However, _**wheel==0.33.4**_ does not satisfy _**wheel==0.24.0**_.

### Dependency tree-------
```
Ops(version range:)
| +-aioredis(version range:==1.1.0)
| +-amqp(version range:==2.3.2)
| +-ansible(version range:==2.4.1.0)
| +-asgi-redis(version range:==1.4.3)
| +-asgiref(version range:==2.3.2)
| +-asn1crypto(version range:==0.24.0)
| +-async-timeout(version range:==3.0.0)
| +-attrs(version range:==18.1.0)
| +-autobahn(version range:==18.5.2)
| +-automat(version range:==0.6.0)
| +-bcrypt(version range:==3.1.4)
| +-billiard(version range:==3.5.0.4)
| | +-wheel(version range:>=0.29.0)
| | +-case(version range:>=1.3.1)
| | +-flake8(version range:>=2.5.4)
| | +-flakeplus(version range:>=1.1)
| | +-pytest(version range:>=3.0)
| | +-pytest-cov(version range:)
| | +-setuptools(version range:>=20.6.7)
| | +-tox(version range:>=2.3.1)
| | +-0(version range:>=0.29.0)
| +-celery(version range:==4.2.0)
| +-certifi(version range:==2018.4.16)
| +-cffi(version range:==1.11.5)
| +-channels(version range:==2.1.1)
| +-channels-redis(version range:==2.2.1)
| +-chardet(version range:==3.0.4)
| +-constantly(version range:==15.1.0)
| +-cryptography(version range:==2.2.2)
| +-daphne(version range:==2.1.2)
| +-diff-match-patch(version range:==20121119)
| +-django(version range:==2.0.5)
| +-django-celery-beat(version range:==1.2.0)
| +-django-celery-results(version range:==1.0.1)
| +-django-cors-headers(version range:==2.5.0)
| +-django-crispy-forms(version range:==1.7.2)
| | +-coverage(version range:)
| | +-django(version range:)
| | +-pytest(version range:)
| | +-pytest-cov(version range:)
| | +-pytest-django(version range:)
| | +-twine(version range:==1.5.0)
| | +-wheel(version range:==0.24.0)
| +-django-filter(version range:==1.1.0)
| +-django-formtools(version range:==2.1)
| | +-Django(version range:>=1.11)
| +-django-haystack(version range:==2.8.1)
| +-django-import-export(version range:==1.0.1)
| +-django-reversion(version range:==2.0.13)
| +-django-timezone-field(version range:==3.0)
| +-djangorestframework(version range:==3.8.2)
| +-et-xmlfile(version range:==1.0.1)
| | +-python(version range:>=2.6.0)
| +-future(version range:==0.15.2)
| +-gevent(version range:==1.3.7)
| +-gitdb2(version range:==2.0.5)
| +-gitpython(version range:==2.1.11)
| +-greenlet(version range:==0.4.15)
| +-hiredis(version range:==0.2.0)
| +-html5lib(version range:==1.0.1)
| +-httplib2(version range:==0.9.2)
| +-hyperlink(version range:==18.0.0)
| +-idna(version range:==2.6)
| +-incremental(version range:==17.5.0)
| +-jdcal(version range:==1.4)
| +-jieba(version range:==0.39)
| +-jinja2(version range:==2.10)
| +-kombu(version range:==4.2.1)
| +-markdown(version range:==2.6.11)
| +-markupsafe(version range:==1.0)
| +-msgpack(version range:==0.5.6)
| +-msgpack-python(version range:==0.5.6)
| +-nose(version range:==1.3.7)
| +-odfpy(version range:==1.3.6)
| +-openpyxl(version range:==2.5.3)
| +-paramiko(version range:==2.4.1)
| +-pillow(version range:==5.2.0)
| +-ply(version range:==3.11)
| +-pyasn1(version range:==0.4.2)
| +-pycparser(version range:==2.18)
| +-pycryptodomex(version range:==3.6.1)
| +-pyguacamole(version range:==0.8)
| +-pymongo(version range:==3.6.1)
| +-pymysql(version range:==0.8.1)
| +-pynacl(version range:==1.2.1)
| +-pysmi(version range:==0.3.0)
| +-pysnmp(version range:==4.4.4)
| +-python-crontab(version range:==2.3.5)
| | +-python-dateutil(version range:)
| +-python-dateutil(version range:==2.7.3)
| +-python-nmap(version range:==0.6.1)
| | +-coverage(version range:==4.0.3)
| | +-flake8(version range:==2.5.1)
| | +-mccabe(version range:==0.3.1)
| | +-nose(version range:==1.3.7)
| | +-pep8(version range:==1.7.0)
| | +-pluggy(version range:==0.3.1)
| | +-pudb(version range:==2015.4.1)
| | +-py(version range:==1.4.31)
| | +-pyflakes(version range:==1.0.0)
| | +-pygments(version range:==2.1)
| | +-tox(version range:==2.3.1)
| | +-urwid(version range:==1.3.1)
| | +-virtualenv(version range:==14.0.1)
| +-pytz(version range:==2018.4)
| +-pyyaml(version range:==3.12)
| +-redis(version range:==2.10.6)
| +-requests(version range:==2.19.1)
| +-six(version range:==1.10.0)
| +-smmap2(version range:==2.0.5)
| +-svn(version range:==0.3.46)
| +-tablib(version range:==0.12.1)
| | +-odfpy(version range:)
| | +-openpyxl(version range:)
| | | +-jdcal(version range:)
| | | +-et_xmlfile(version range:)
| | +-unicodecsv(version range:)
| | +-xlrd(version range:)
| | +-xlwt(version range:)
| | +-pyyaml(version range:)
| +-twisted(version range:==17.5.0)
| +-txaio(version range:==2.10.0)
| +-unicodecsv(version range:==0.14.1)
| +-urllib3(version range:==1.23)
| +-vine(version range:==1.1.4)
| +-webencodings(version range:==0.5.1)
| +-websockets(version range:==5.0.1)
| +-whoosh(version range:==2.7.4)
| +-xlrd(version range:==1.1.0)
| +-xlwt(version range:==1.3.0)
| +-zope.interface(version range:==4.5.0)
```

Thanks for your help.
Best,
Neolith

**Comment by NeolithEra** [comment]
### Solution

1.	Fix your direct dependency to be **_wheel==0.24.0_**. I have checked this revision will not affect your downstream projects now.

2.	Ask your upstream project **_django-crispy-forms_** to loose the version range of wheel.

Which solution do you prefer, 1 or 2?
@pythonzm Please let me know your choice. I can submit a PR to solve this issue.

**Comment by pythonzm** [comment]
i don't think it's a problem
